tests/cli.rs

```diff
@@ -1191,6 +1195,58 @@ fn cert_without_key() {
         .stderr(predicates::str::is_empty());
 }
 
+#[cfg(feature = "online-tests")]
+#[test]
+fn use_ipv4() {
+    get_command()
+        .args(&["https://v6.ipv6-test.com/api/myip.php", "--body", "--ipv4"])
+        .assert()
+        .failure();
+
+    get_command()
+        .args(&[
+            "https://v4v6.ipv6-test.com/api/myip.php",
+            "--body",
+            "--ipv4",
+        ])
+        .assert()
+        .stdout(function(|output: &str| {
+            IpAddr::from_str(output.trim()).unwrap().is_ipv4()
+        }))
+        .stderr(predicates::str::is_empty());
```

Comment:
This test seems to fail randomly. Can we have it behind a feature flag as well?

Reply:
may be wo can use another api to test ipv4/ipv6, such as https://www.ipify.org/